api/src/test/java/org/pmiops/workbench/actionaudit/ActionAuditQueryServiceTest.java

```diff
@@ -97,9 +97,9 @@ public class ActionAuditQueryServiceTest {
   private static final Page<FieldValueList> WORKSPACE_QUERY_RESULT_PAGE =
       new FakeSinglePage<>(WORKSPACE_RESULT_ROWS);
   private static final TableResult WORKSPACE_TABLE_RESULT =
-      new TableResult(
```

Comment:
this constructor and EmptyTableResult were removed from more recent versions of the BQ client, so I added BigQueryUtils to simplify the upgrade.